pkg/tests/apis/query/query_test.go

```diff
@@ -137,17 +137,26 @@ func TestIntegrationSimpleQuery(t *testing.T) {
 		body, err = result.Raw()
 		//fmt.Printf("OUT: %s", string(body))
 
-		require.Error(t, err, "expecting a 400")
+		require.Error(t, err, "expecting an error")
 		require.JSONEq(t, `{
-			"status": "Failure",
+			"kind": "Status",
+			"apiVersion": "v1",
 			"metadata": {},
+			"status": "Failure",
 			"message": "did not execute expression [Y] due to a failure to of the dependent expression or query [X]",
-			"reason": "BadRequest",
-			"details": { "group": "query.grafana.app" },
-			"code": 400,
-			"messageId": "sse.dependencyError",
-			"extra": { "depRefId": "X", "refId": "Y" }
+			"reason": "Bad request",
+			"code": 400
 		  }`, string(body))
+		// require.JSONEq(t, `{
+		// 	"status": "Failure",
+		// 	"metadata": {},
+		// 	"message": "did not execute expression [Y] due to a failure to of the dependent expression or query [X]",
+		// 	"reason": "BadRequest",
+		// 	"details": { "group": "query.grafana.app" },
+		// 	"code": 400,
+		// 	"messageId": "sse.dependencyError",
+		// 	"extra": { "depRefId": "X", "refId": "Y" }
```

Comment:
these are from errutil -- but don't have an easy way to map into metav1.Status 🤔 

In general, we need error handling to be compatible -- but that is a bigger project than this PR!